### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by moigagoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,17 +44,17 @@ Nim is an Open Source programming language in which many Open Source projects ar
...
-**First name**
+**First name** Pietro
 
-**Last name**
+**Last name** Peterlongo
 
 **E-mail**: *we can skip this in this repo*
 
-**Additional pentabarf ids:** pietroppeter, federico3
+**Additional pentabarf ids:** pietroppeter, federico3, moigagoo, xigoi
```
Comment (pietroppeter) [inline README.md:57]: did you create your pentabarf account using fosdem system (https://penta.fosdem.org/user/new_account)?
do you know if @xigoi has created a pentabarf account with same user of GitHub or forum?

**Review by pietroppeter — COMMENTED** [review]
I think this should be closed, but thanks for trying to tick all boxes :). If you can confirm you have created a pentabarf account we should add it to the list

File: README.md (hunk 1 above)
Comment (moigagoo) [reply README.md:57]: No problem!

> If you can confirm you have created a pentabarf account we should add it to the list

How do I get it? Googling doesn't help much.

Comment (pietroppeter) [reply README.md:57]: through the link I posted in the comment 😉: https://penta.fosdem.org/user/new_account

**Comment by pietroppeter** [comment]
posting the link to get pentabarf account (for @moigagoo) also here (not sure how much comments in code notify): https://penta.fosdem.org/user/new_account

File: README.md (hunk 1 above)
Comment (moigagoo) [reply README.md:57]: > through the link I posted in the comment 😉: https://penta.fosdem.org/user/new_account

Sorry, I missed it.

> did you create your pentabarf account using fosdem system (https://penta.fosdem.org/user/new_account)?

I've created the account. The account name is moigagoo.

Comment (xigoi) [reply README.md:57]: I've created my account now.

**Pull request closed without merging**